sklearn/ensemble/gradient_boosting.py

```diff
@@ -195,16 +195,23 @@ def update_terminal_regions(self, tree, X, y, residual, y_pred,
         ----------
         tree : tree.Tree
             The tree object.
-        X : np.ndarray, shape=(n, m)
+        X : ndarray, shape=(n, m)
             The data array.
-        y : np.ndarray, shape=(n,)
+        y : ndarray, shape=(n,)
             The target labels.
-        residual : np.ndarray, shape=(n,)
+        residual : ndarray, shape=(n,)
             The residuals (usually the negative gradient).
-        y_pred : np.ndarray, shape=(n,):
+        y_pred : ndarray, shape=(n,):
             The predictions.
-        sample_weight : np.ndarray, shape=(n,):
+        sample_weight : ndarray, shape=(n,):
             The weight of each sample.
+        sample_mask : ndarray
+            The sample mask to be used.
```

Comment:
`masked` was equal to `unmasked[~sample_mask]` so I think the opposite... Should it be documented as `Use inverted logic to specify the samples to be used` ?